apiclient/harvester_api/managers.py

```diff
@@ -451,8 +451,8 @@ def get(self, name="", namespace=DEFAULT_NAMESPACE, *, raw=False):
         return self._get(path, raw=raw)
 
     def create(self, name, vlan_id, namespace=DEFAULT_NAMESPACE, *,
-               cluster_network=None, raw=False):
-        data = self.create_data(name, namespace, vlan_id, self._bridge_name(cluster_network))
+               cluster_network=None, mode="auto", cidr="", gateway="", raw=False):
+        data = self.create_data(name, namespace, vlan_id, self._bridge_name(cluster_network), mode=mode, cidr=cidr, gateway=gateway)
```

Comment:
the maximum length is 99, please fix it like:
```python
data = self.create_data(name, namespace, vlan_id, self._bridge_name(cluster_network),
                        mode=mode, cidr=cidr, gateway=gateway)
```

then LGTM.

cc @khushboo-rancher